packages/react-components/react-tree/src/components/Tree/useTree.ts

```diff
@@ -38,7 +38,7 @@ function useNestedRootTree(props: TreeProps, ref: React.Ref<HTMLElement>): TreeS
         }),
         onNavigation: useEventCallback((event, data) => {
           props.onNavigation?.(event, data);
-          if (!event.isDefaultPrevented()) {
+          if (!event.defaultPrevented) {
```

Comment:
```suggestion
          // isDefaultPrevented cannot be used here
          // as we're creating a synthetic event
          // on element unmount
          if (!event.defaultPrevented) {
```
nit: Can you add a comment here explaining why we cannot trust in `isDefaultPrevented` anymore with this change?!